net-vpn/protonvpn-cli/protonvpn-cli-2.2.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A VPN command-line tool from protonvpn - python rewrite"
+HOMEPAGE="(
```

Comment:
This on the other hand is not an array. You can just place both links in the same line inside `"..."`.